web/src/engine/websites/PocketComics_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture } from '../../../test/WebsitesFixture';
+
+const config = {
+    plugin: {
+        id: 'pocketcomics',
+        title: 'Pocket-Comics (コミコ)'
+    },
+    container: {
+        url: 'https://www.pocketcomics.com/comic/10040',
+        id: JSON.stringify({ id: '/comic/10040', lang: 'en-US' }),
+        title: 'The Devilishly Trash Duke'
+    },
+    child: {
+        id: '1',
+        title: 'Prologue'
+    },
+    entry: {
+        index: 0,
+        size: 184_838,
+        type: 'image/jpeg'
+    }
+};
+
+const fixture = new TestFixture(config);
+describe(fixture.Name, () => fixture.AssertWebsite());
```

Comment:
I wanted too but its impossible. On pasting, langage is set by langage setting of plugin,.

* API need a langage to fetch manga info
* Same manga id (/comic/12345/) can lead to the french eng or cn manga depending on the API CALL.
* Therefore in pasting lang is set arbitrarily

Reply:
This is just the case when using the `WebsiteFixture`.
It would be possible to write the corresponding puppeteer tests manually, however not sure if it is worth the effort.

Reply:
May sync with `master` branch and use the new settings configuration
```typescript
import { type Config, TestFixture } from '../../../test/WebsitesFixture';

const configEnglish: Config = {
    plugin: {
        id: 'pocketcomics',
        title: 'Pocket-Comics (コミコ)',
        settings: {
            language: 'en-US'
        }
    },
    container: {
        url: 'https://www.pocketcomics.com/comic/10040',
        id: JSON.stringify({ id: '/comic/10040', lang: 'en-US' }),
        title: 'The Devilishly Trash Duke'
    },
    child: {
        id: '2',
        title: 'Episode 1'
    },
    entry: {
        index: 0,
        size: 114_618,
        type: 'image/jpeg'
    }
};

const fixtureEnglish = new TestFixture(configEnglish);
describe(fixtureEnglish.Name, () => fixtureEnglish.AssertWebsite());

const configFrench: Config = {
    plugin: {
        id: 'pocketcomics',
        title: 'Pocket-Comics (コミコ)',
        settings: {
            language: 'fr-FR'
        }
    },
    container: {
        url: 'https://www.pocketcomics.com/comic/10040',
        id: JSON.stringify({ id: '/comic/10040', lang: 'fr-FR' }),
        title: 'Un duc diaboliquement infâme'
    },
    child: {
        id: '2',
        title: 'Épisode 1'
    },
    entry: {
        index: 0,
        size: 105_584,
        type: 'image/jpeg'
    }
};

const fixtureFrench = new TestFixture(configFrench);
describe(fixtureFrench.Name, () => fixtureFrench.AssertWebsite());
```

Reply:
awesome